Automattic/wp-calypso
# Stats: Empty state for the page chart

Related to https://github.com/Automattic/red-team/issues/79 p1722387378151689/1722215313.108939-slack-C04UE0ANHDY

## Proposed Changes

* adding an icon to the notice visible when the main chart has no data

## Why are these changes being made?


* it's part of the empty states project

## Testing Instructions



* open the live branch
* navigate to a period when there is no data available on the Traffic page (e.g. past dates for a JN site)
* verify that the notice matches the screenshot below
* repeat for post details page

<img width="1264" alt="SCR-20240801-jmjm" src="https://github.com/user-attachments/assets/ef3844f4-aa71-439a-b085-a0695057fcb1">


## Pre-merge Checklist
